gb-cpu/src/bus.rs

```diff
@@ -0,0 +1,8 @@
+pub trait Bus<T> {
```

Comment:
I don't understand this trait, consider renaming it and adding a command explaining why (using `///`)

Reply:
This trait is the equivalent of your FileOperation